src/components/AppHeader.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+export const AppHeader = styled.header`
+  background-color: #222;
```

Comment:
move to some common variables

Reply:
You mean as [theme](https://www.styled-components.com/docs/api#typescript) based variables?

Reply:
yes